ios/BlurView.m

```diff
@@ -37,12 +37,13 @@ - (void)setBlurType:(NSString *)blurType
     } else {
         self.blurEffect = [UIBlurEffect effectWithStyle:UIBlurEffectStyleDark];
```

Comment:
@haywoodfu - I opened different PR to fix this as well (#183).

> As far as right now, i can't resolve it without category...

I solved this with "associated objects", so I can set a `blurAmount` on each instance of the UIVisualEffect.